docs/kafka/kafka-bin-scripts/README.adoc

```diff
@@ -141,44 +141,48 @@ You can use the `kafka-console-producer` script to produce messages to Kafka top
 
 * You have a running Kafka instance in {product}.
 ifndef::qs[]
-* You have the bootstrap server endpoint for your Kafka instance. To relocate the server endpoint, select your Kafka instance in the {product} web console, select the options menu (three vertical dots), and click *Connection*.
-* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, select your Kafka instance in the {product} web console and use the *Access* page to find your service account permission settings.
+* You have the bootstrap server endpoint for your Kafka instance. To get the server endpoint, select your Kafka instance in the {product} web console. Select the options menu (three vertical dots), and click *Connection*.
+* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, select your Kafka instance in the
+{product} web console and use the *Access* page to find your service account permission settings.
 endif::[]
 * You've created the `{property-file-name}` file to store your service account credentials.
 
 .Procedure
 . On the command line, from the `bin/` directory, enter the following command to create a Kafka topic.
 +
 --
-This example uses the `kafka-topics` script to create the `my-other-topic` Kafka topic with the default settings.
 
 ifdef::qs[]
 The `<bootstrap_server>` is the bootstrap server endpoint for your Kafka instance. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots) and clicking *Connection*.
 endif::[]
 
-.Using the `kafka-topics` script to create a Kafka topic
+Using the `kafka-topics` script to create a Kafka topic
```

Comment:
This might be accidental - I think you still want the preceding period so that the text is styled as a title for the code block.
```suggestion
.Using the `kafka-topics` script to create a Kafka topic
```